src/pubsub.js

```diff
@@ -122,7 +122,7 @@ module.exports = (common) => {
               expect(msg.data.toString()).to.equal('hi')
               expect(msg).to.have.property('seqno')
               expect(Buffer.isBuffer(msg.seqno)).to.be.eql(true)
-              expect(msg).to.have.property('topicCIDs').eql([topic])
+              expect(msg).to.have.property('topicIDs').eql([topic])
```

Comment:
This means that js-ipfs-api is also incorrect. Mind checking that?

Reply:
Good call, already fixed in https://github.com/ipfs/js-ipfs-api/pull/627